src/Backend/OnboardingInsuranceAPI/Areas/Insurance/GetContribution.cs

```diff
@@ -20,24 +14,34 @@ public GetContribution(DataContext context)
         _context = context;
     }
 
-    public async Task<ContributionDataResponse> Handle(ContributionDataRequest requestedData, string pid)
+    public ContributionDataResponse Handle(ContributionDataRequest requestedData, string pid)
```

Comment:
Why it's not Async anymore? You are still using _context. You should access things asynchronously whenever possible.

Reply:
Ok, this method do not do any async things so I decided to remove it. I'll return it to async.